server/http.go

```diff
@@ -28,8 +28,13 @@ const (
 	zoomOAuthUserStateLength = 4
 	settingDataError         = "something went wrong while getting settings data"
 	askForPMIMeeting         = "Would you like to use your personal meeting ID?"
-	apiToUpdatePMI           = "/plugins/%s/api/v1/updatePMI"
-	apiToAskForPMI           = "/plugins/%s/api/v1/askPMI"
+	pathWebhook              = "/webhook"
+	pathStartMeeting         = "/api/v1/meetings"
```

Comment:
APIs starting with path ```/api/v1/``` have been added by you or did they exist before hand as well?

Reply:
It exits earlier as well https://github.com/mattermost/mattermost-plugin-zoom/blob/8e898623befc730f7a161ff15a60a8ffe5e28433/server/http.go#L48